### PR TITLE
Document how `to-ascii` transforms characters

### PR DESCRIPTION
**Pull request by 9999years**
I was confused and unsure what the `rnr to-ascii` command would do, so I looked up the implementation and added some details and examples:

```
$ cargo run --help
rnr-to-ascii
Replaces UTF-8 chars in file names with approximate ASCII equivalents.

Non-ASCII Unicode characters are replaced with ASCII characters in an ad-hoc
fashion. Here are some example transformations:

άνθρωποι -> anthropoi
深圳     -> ShenZhen
Борис    -> Boris
トヨタ   -> toyota
👑       -> :crown:
♯        -> #

Unknown characters and some known characters are replaced with an empty string
and removed. These include (but are not limited to) non-ASCII control
characters like U+0080, Private Use Area codepoints like U+E000, and
noncharacters like U+FFFF.
```

### CODE REVIEW

**Comment by ismaelgv** [comment]
@9999years thanks for the contribution.

It would be nice to extend the README of the project to match this information.

First, I would reword this documentation a bit too. The note "in an ad-hoc fashion" is quite confusing and it seems to imply that the replacement is non-deterministic or depends in the context.

**Comment by 9999years** [comment]
@ismaelgv What language would you prefer? I wanted to clarify that the conversion is not done with respect to any published Unicode standard, and is instead performed ["for the particular end or case at hand without consideration of wider application"](https://www.merriam-webster.com/dictionary/ad%20hoc).

**Comment by ismaelgv** [comment]
We can just document the source of the transliteration [AnyAscii](https://github.com/anyascii/anyascii). There are more examples there and detailed information of the conversion.

About the _ad hoc_ expression, it is not based in an official Unicode standard, but it is not particular conversion for this context or application. It is based in Unidecode and other sources, as stated in the library documentation. Hence, it is actually applied in many other projects and not just here.

File: src/app.rs [hunk 1]
```diff
@@ -128,7 +128,25 @@ pub fn create_app<'a>() -> App<'a, 'a> {
             SubCommand::with_name(TO_ASCII_SUBCOMMAND)
                 .args(&common_args)
                 .args(&path_args)
-                .about("Replace file name UTF-8 chars with ASCII chars representation."),
+                .about("Replace UTF-8 chars in file names with approximate ASCII equivalents.")
+                .long_about(
+                    "Replaces UTF-8 chars in file names with approximate ASCII equivalents.\n\
+                    \n\
+                    Non-ASCII Unicode characters are replaced with ASCII characters in an ad-hoc\n\
+                    fashion. Here are some example transformations:\n\
```
Comment (ismaelgv) [inline src/app.rs:136]: suggestion: Here we can reference AnyAscii directly to provide a source where the user can find the actual behavior and conversion tables.

**Review by ismaelgv — CHANGES_REQUESTED** [review]
Thanks again for the contribution @9999years. We can change that sentence and add this documentation also to README.md.

**Comment by ismaelgv** [comment]
`master` has been migrate to `main`

**Comment by ismaelgv** [comment]
`master` has been migrate to `main`

**Comment by 9999years** [comment]
> We can just document the source of the transliteration [AnyAscii](https://github.com/anyascii/anyascii). There are more examples there and detailed information of the conversion.

That sounds like a good solution to me.

Do you want me to reopen this? (If I understand correctly, GitHub closes open PRs when you delete the branch they're targeting, so I'm not sure if this was a "this PR is no longer needed" close or a "the base branch is gone" close.)

**Comment by ismaelgv** [comment]
This has been closed by GitHub after deleting `master` branch. Feel free to open a new pull request to `main`.
